e2e/go.mod

```diff
@@ -1,4 +1,4 @@
-module e2e
+module github.com/cosmos/ibc-go/e2e
```

Comment:
would like to have @chatton approval on this change. Is it okay to include it here or should we do it in a separate pr. I guess it doesn't need to be backported so it doesn't matter?

Reply:
This change is fine to have here 👍